### PR TITLE
Add new Artifacts Finder plugin and Fix Artifactory issue

### PR DESCRIPTION
**Pull request by snowtigersoft**
1. Add a new Artifacts Finder plugin, which can auto prospect and find artifacts.
2. Fix Artifactory issue, to make it works on 0.6

### CODE REVIEW

File: content/artifacts/artifacts-finder/plugin.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
+// Artifacts Finder
+// This plugin can help auto prospect and find artifacts.
+// Simple is Power!
+
```
Comment (jacobrosenthal) [inline content/artifacts/artifacts-finder/plugin.js:4]: Maybe mention start finding runs on a 10 minute timer in the background, and the only way to stop finding is to close the plugin.

File: content/artifacts/artifacts-finder/plugin.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
+// Artifacts Finder
+// This plugin can help auto prospect and find artifacts.
```
Comment (jacobrosenthal) [inline content/artifacts/artifacts-finder/plugin.js:2]:
```suggestion
// This plugin offers a button to auto prospect and find artifacts every 10 minutes. To stop simply close the plugin
```

File: content/artifacts/artifacts-finder/plugin.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+// Author: SnowTiger
+// If you feel this plugin is helpful, you can transfer a small donation
+// to this xDai Account: 0xaa23208770da9ae738a7a5069bcdfdc06e487821
+
+const {
+    BiomeNames,
+    energy,
+    coords,
+    canHaveArtifact,
+} = await import('https://plugins.zkga.me/utils/utils.js');
```
Comment (phated) [inline content/artifacts/artifacts-finder/plugin.js:14]: Wrong imports style. Doesn't work in all browsers.

File: content/artifacts/artifactory/plugin.js [hunk 4]
```diff
@@ -7,537 +7,608 @@
...
+    canHaveArtifact,
+} = await import('https://plugins.zkga.me/utils/utils.js');
+
+const {
+    Energy,
+    EnergyGrowth,
+    Defense,
+    Range,
+    Speed,
+} = await import('https://plugins.zkga.me/game/Icons.js');
```
Comment (phated) [inline content/artifacts/artifactory/plugin.js:35]: This is the wrong import style. I just went through and updated all of these. Can't use this and expect to work on all browsers.

**Comment by phated** [comment]
@snowtigersoft when you update plugins in the future. Please don't change the formatting. It makes the diff impossible to review with any accuracy.

File: content/artifacts/artifacts-finder/plugin.js (hunk 1 above)
Comment (snowtigersoft) [reply content/artifacts/artifacts-finder/plugin.js:4]: When start finding, the button change to "Cancel Finding", you can stop finding by click button, or close the button. The different is click close button can keep the finding log.

File: content/artifacts/artifactory/plugin.js (hunk 4 above)
Comment (snowtigersoft) [reply content/artifacts/artifactory/plugin.js:35]: Understood now, thank you.

**Pull request merged**